StantUiIosLibrary/Classes/Dialogs/BottomDialogWithScroll/BottomDialogWithScrollViewController.swift

```diff
@@ -0,0 +1,194 @@
+        guard let mainView = mainView else { return }
+        view.layer.cornerRadius = 3.5
+        view.backgroundColor    = UIColor.lightGrayStant
+        
+        view.anchor(top: mainView.topAnchor,
+                    padding: UIEdgeInsets(top: 16, left: 0, bottom: 0, right: 0),
+                    size: CGSize(width: 120, height: 6))
+        view.anchorCenterX(anchorX: mainView.centerXAnchor)
+    }
+    
+    fileprivate func configureTableView() {
+        guard let mainView = mainView, let tableView = tableView else { return }
+        
+        tableView.delegate                 = self
+        tableView.dataSource               = self
+        tableView.separatorStyle           = .none
+        tableView.isUserInteractionEnabled = true
+        tableView.isScrollEnabled          = true
+        tableView.allowsMultipleSelection  = false
+        
+        tableView.register(ScrollableTableViewDialogCell.self, forCellReuseIdentifier: ScrollableTableViewDialogCell.IDENTIFIER)
+        
+        tableView.anchor(top: mainView.topAnchor,
+                         leading: mainView.leadingAnchor,
+                         bottom: mainView.bottomAnchor,
+                         trailing: mainView.trailingAnchor,
+                         padding: UIEdgeInsets(top: 38, left: 0, bottom: 0, right: 0))
+    }
+    
+}
+
+extension BottomDialogWithScrollViewController: UITableViewDelegate, UITableViewDataSource {
+    
+    public func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return items.count
+    }
+    
+    public func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return ScrollableTableViewDialogCell.cellHeight
+    }
+    
+    public func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: ScrollableTableViewDialogCell.IDENTIFIER) as? ScrollableTableViewDialogCell else {
+            return UITableViewCell()
+        }
+        cell.selectionStyle = .none
+        cell.configureViewWith(title: items[indexPath.row])
+        
+        if indexPath.row == selectedItemIndex {
+            cell.accessoryType = .checkmark
+        }
+        
+        return cell
+    }
+    
+    public func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        self.dismissScreen()
+
+        if selectedItemIndex != indexPath.row {
+            selectTableViewCellAt(indexPath: indexPath)
+        }
+    }
+    
+    fileprivate func selectTableViewCellAt(indexPath: IndexPath) {
+        let cell = tableView(tableView ?? UITableView(), cellForRowAt: indexPath) as? ScrollableTableViewDialogCell
+        cell?.accessoryType = .checkmark
+        
+        cellDelegate?.clickOnCellWith(index: indexPath.row, title: cell?.cellTextLabel?.text ?? "")
+        
+        for i in 0...(items.count - 1) {
```

Comment:
You could do a enumerated loop, is more readable: 
`for (index, element) in list.enumerated() {
  print("Item \(index): \(element)")
}`

Reply:
Done. This is a new way to treat loop. Cool